packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -77,6 +77,7 @@ class HookshotContent extends React.Component {
 
   handleClickOutside(event) {
     if (this.props.onOutsideClick) {
+      event.preventDefault();
```

Comment:
is there a way to add a test for this?

Reply:
I think the only thing you could really add is a jest test, and verify that preventDefault was called when handleClickOutside is triggered.

Reply:
To verify this is working as expected do the following:


1. Start up a mobile device or the simulator. Go to the http://engineering.cerner.com/terra-core/#/site/popup and click around on the popup attachments buttons. Notice the behavior is to continuously trigger the next popup. 
- If you check this on a desktop browser, the behavior is the popup will close and an additional click is needed to trigger the next action on the page.
- This is the bug in # 1003

2. Now on the simulator, go to https://terra-core-deployed-pr-1029.herokuapp.com/static/#/site/popup. The behavior is now consistent with the desktop browser behavior. 
- If you check the desktop browser, the behavior remains the same.